src/component/axis/Axis.jsx

```diff
@@ -2,6 +2,29 @@
 
 import React from 'react';
 
+function DefaultAxisLine(props: Object): React.Element<any> {
```

Comment:
Not directly relating to this line but we need to document somewhere what props get passed to each of the custom components.